core/src/main/java/io/ultrabrew/metrics/util/Commons.java

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019, Oath Inc.
+// Licensed under the terms of the Apache License 2.0 license. See LICENSE file in Ultrabrew Metrics
+// for terms.
+
+package io.ultrabrew.metrics.util;
+
+public class Commons {
+
+  public static void checkArgument(final boolean condition, final String message) {
```

Comment:
This method (and/or the class) could use better name, it would be nice to see straight away if it throws if the condition is `true`/`false` without reading code or javadoc.

Reply:
i.e. "assertTrue"